src/lightning/pytorch/strategies/xla_fsdp.py

```diff
@@ -247,28 +245,21 @@ def process_dataloader(self, dataloader: object) -> "MpDeviceLoader":
         return dataloader
 
     def setup_optimizers(self, trainer: "pl.Trainer") -> None:
-        invalid_params_error = False
-        try:
-            super().setup_optimizers(trainer)
```

Comment:
@carmocca why do we remove `super().setup_optimizers(trainer)`? I notice that once we removed this line, the optimizer is never setup, so `self.optimizers = []`